meson.build

```diff
@@ -0,0 +1,21 @@
+# #############################################################################
+# Copyright (c) 2018-present    lzutao <taolzu(at)gmail.com>
+# All rights reserved.
+#
+# This source code is licensed under both the BSD-style license (found in the
+# LICENSE file in the root directory of this source tree) and the GPLv2 (found
+# in the COPYING file in the root directory of this source tree).
+# #############################################################################
+
+# This is the dummy meson file from upstream, with version and path adjusted.
+
+project('lz4', ['c'],
+  license: ['BSD', 'GPLv2'],
+  default_options : ['c_std=c99',
+    'buildtype=release'],
+  version: '1.9.2',
+  meson_version: '>=0.47.0')
+
+subdir('contrib/meson/meson')
+
+lz4_dep = liblz4_dep # Not sure this is a good idea…
```

Comment:
We usually have a style requirement of having `${wrap}_dep` exported, so I think it is.

Reply:
Ty. *edited comment*